BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -85,8 +85,10 @@ class WalletViewController: UIViewController {
       dollarValue: state.ledger.usdBalanceString
     )
     
+    let now = Date()
+    rewardsSummaryView.monthYearLabel.text = "\(now.currentMonthName().uppercased()) \(now.currentYear)"
+    
     // FIXME: Remove temp values
-    rewardsSummaryView.monthYearLabel.text = "MARCH 2019"
```

Comment:
Ah! Almost forgot there is a second usage of this 😆 Sorry!

https://github.com/brave/brave-rewards-ios/blob/a79aabdd8c878a89bfef03618c036b08b26b7c1d/BraveRewardsUI/Settings/Wallet/WalletDetailsViewController.swift#L44

Reply:
Went with a slightly bigger change and put some shared code under a `protocol` to not miss it again. Lmk if that's too much of a refactor